### PR TITLE
feat(crux): add Prometheus basics

### PR DESCRIPTION
**Pull request by polaroi8d**
Add Prometheus with an agent connection counter.

### CODE REVIEW

File: web/crux/src/app/agent/agent.service.ts [hunk 1]
```diff
@@ -302,6 +305,7 @@ export default class AgentService {
     connection.status().subscribe(it => this.onAgentConnectionStatusChange(agent, it))
 
     this.logger.log(`Agent joined with id: ${request.id}, key: ${!!agent.publicKey}`)
+    this.agent_counter.inc()
```
Comment (robot9706) [inline web/crux/src/app/agent/agent.service.ts:308]: Does this decrease when an agent disconnects?

**Review by robot9706 — CHANGES_REQUESTED** [review]

File: web/crux/src/app/agent/agent.service.ts (hunk 1 above)
Comment (polaroi8d) [reply web/crux/src/app/agent/agent.service.ts:308]: Nope, this is a counter function that represents a single increasing counter whose value can only increase or be reset to zero on restart.

**Review by robot9706 — APPROVED** [review]
LGTM, Tested, metrics endpoint seems to be working properly 🚀

**Review by peterambrus3 — APPROVED** [review]
lgtm, tested locally

**Pull request merged**